apps/src/templates/rubrics/rubrics.module.scss

```diff
@@ -195,16 +195,11 @@ p.aiTokenText {
   color: $product_affirmative_default;
 }
 
-.expandedBorder {
-  border-top: 1px solid $neutral_dark40;
-}
-
 .learningGoalExpanded {
-  padding: 1.25rem 1rem;
-
+  padding: 1rem;
   display: flex;
-  flex-direction: row;
-  gap: 2rem;
+  flex-direction: column;
```

Comment:
I checked in with Mark about this.  Initially Bethany had asked that we wait on fixing this, but it was hard to wait with this PR.  Mark seemed to think it was OK.